fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -257,7 +257,7 @@ module.exports = function(config, DB) {
               })
               .then(function (sessions) {
                 t.equal(sessions.length, 1, 'sessions contains one item')
-                t.equal(Object.keys(sessions[0]).length, 9, 'session has nine properties')
+                t.equal(Object.keys(sessions[0]).length, 16, 'session has correct properties')
```

Comment:
I think it might be useful to have a test case where you create a session with a device and verify that correct device properties are set on the session. You could put that into the [accountDevices](https://github.com/mozilla/fxa-auth-db-mysql/blob/b467014aadf5d46e8dbf8209abfe923de869e72b/fxa-auth-db-server/test/backend/db_tests.js#L1077) test section to help reuse code.